example/hello_world.py

```diff
@@ -1,54 +1,110 @@
 #!/usr/bin/env python3
 import houston
 import bugzoo
+from bugzoo.core.coverage import TestCoverage, TestSuiteCoverage
+from bugzoo.core.spectra import Spectra
+from bugzoo.localization import Localization
+from bugzoo.localization.suspiciousness import tarantula
 import pprint
 from houston.generator.rand import RandomMissionGenerator
 from houston.generator.resources import ResourceLimits
+import copy
 
-#bz = bugzoo.BugZoo()
-#snapshot = bz.bugs['ardudemo:ardupilot:overflow']
-#snapshot.build()
-
-sut = houston.ardu.ArduRover('afrl:overflow')
-
-# mission description
-actions = [
-    houston.action.Action("arm", {'arm': True}),
-    houston.action.Action("goto", {
-        'latitude' : -35.361354,
-        'longitude': 149.165218,
-        'altitude' : 5.0
-    })
-]
-environment = houston.state.Environment({})
-initial = houston.state.State({
-    "homeLatitude" : -35.3632607,
-    "homeLongitude" : 149.1652351,
-    "latitude" : -35.3632607,
-    "longitude": 149.1652351,
-    "altitude" : 0.0,
-    "battery"  : 100,
-    "armed"    : False,
-    "armable"  : True,
-    "mode"     : "AUTO"
-}, 0.0)
-mission = houston.mission.Mission(environment, initial, actions)
-
-# create a container for the mission execution
-sandbox = sut.provision()
-#res = sandbox.run(mission)
-#print(res)
-(res, coverage) = sandbox.run_with_coverage(mission, { "APMrover2/APMrover2.cpp" })
-print("Done")
-pprint.pprint(coverage.to_dict())
-
-# mission_generator = RandomMissionGenerator(sut, initial, environment)
-# resource_limits = ResourceLimits(2, 1000)
-# mission_generator.generate(100, resource_limits)
-#mission_generator.prepare(100, resource_limits)
-#for i in range(5):
-#    m = mission_generator.generate_mission()
-#    print("Mission {}: {}".format(i, m.to_json()))
-#    res = sandbox.run(m)
-#    print(res)
-sandbox.destroy()
+
+#### Run a single mission
+def run_single_mission(sandbox, mission):
+    res = sandbox.run(mission)
+    print(res)
+
+#### Run a single mission with coverage
+def run_single_mission_with_coverage(sandbox, mission):
+    (res, coverage) = sandbox.run_with_coverage(mission, { "APMrover2/APMrover2.cpp" })
```

Comment:
We don't need to pass `files_to_instrument` any longer, so long as the BugZoo manifest has a `coverage.files-to-instrument` property.

Reply:
@ChrisTimperley So does `afrldemo` have the property?
What should I call from bugzoo to configure and recompile with coverage flags?

Reply:
You shouldn't need to change any of the method calls, you'll just want to drop the optional `files_to_instrument` argument. I don't think it does, but it's pretty straightforward to add.

```
coverage:
  files-to-instrument:
    - APMrover2/APMrover2.cpp
    - ArduCopter/ArduCopter.cpp
    - ArduPlane/ArduPlane.cpp
```